x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml

```diff
@@ -119,6 +122,60 @@ setup:
       indices.get:
         index: test
 
+  - do:
```

Comment:
It's good to have those tests in the rollups code, but perhaps it would be simpler to test `aggregate_metric_double` field in isolation at the more [basic aggregate-metrics yaml tests](https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/aggregate-metrics/10_basic.yml)